dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/publish/servers/servlet40_headers/bootstrap.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2021, 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
```

Comment:
clean up Contributors and year...but not necessary.

Reply:
I cleaned up the contributors but leaving dates alone since we're not making updates to the majority of these files they are just being copied over from an existing bucket to this new bucket.